.gitignore

```diff
@@ -5,6 +5,7 @@ chkjson*
 data/*.template
 templates/
 save/
+mods/
```

Comment:
Why? Some mods are part of the baseline.

Reply:
It doesn't ignore baseline mods in data/mods, only user mods.

Reply:
Yes, sorry.